src/compiled/SurveyRunItemView.js

```diff
@@ -404,17 +405,18 @@ SurveyRunItemView = (function(superClass) {
   };
 
   SurveyRunItemView.prototype.onShow = function() {
-    var displayCode, error, error1, message, name, ref;
+    var displayCode, displaycodeFixed, error, error1, message, name, ref;
     displayCode = this.model.getString("displayCode");
     if (!_.isEmptyString(displayCode)) {
+      displaycodeFixed = displayCode.replace("vm.currentView.subtestViews[vm.currentView.index].prototypeView", "Tangerine.progress.currentSubview");
```

Comment:
Yeah, we may want to pop this out into a separate file that indicates its function ("backwards_compatability.json perhaps?) and document the heck out of it ;-)